### PR TITLE
Publish oVirt images

### PR DESCRIPTION
**Pull request by rgolangh**
Publishing 2 oVirt platform images:
- ovirt/cluster-api-provider-ovirt by the name **cluster-api-provider-ovirt**
- ovirt/csi-driver by the name **ovirt-csi-driver**

Enhancement: https://github.com/openshift/enhancements/pull/61

### CODE REVIEW

**Comment by smarterclayton** [comment]
These names are fine, please run the generators as described in the failing tests

**Comment by enxebre** [comment]
/lgtm
@smarterclayton / @petr-muller PTAL

File: ci-operator/config/openshift/cluster-api-provider-ovirt/OWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+approvers:
+  - rgolangh
+reviewers:
+  - rgolangh
```
Comment (petr-muller) [inline ci-operator/config/openshift/cluster-api-provider-ovirt/OWNERS]: Preferrably have at least one more reviewer; who would review **your** PRs?

**Comment by rgolangh** [comment]
its @bennyz and I asked him to join openshift organization.

**Comment by enxebre** [comment]
@rgolangh I think we also need to add the jobs for `openshift-cluster-api-provider-ovirt-release-4.4-*` and `openshift-cluster-api-provider-ovirt-release-4.5-*`

@petr-muller does it need to be added in this PR/followup or is this something your team does automatically along with fast forward from master into release-specific branches.

**Comment by rgolangh** [comment]
> @rgolangh I think we also need to add the jobs for `openshift-cluster-api-provider-ovirt-release-4.4-*` and `openshift-cluster-api-provider-ovirt-release-4.5-*`

I'm adding another commit in this PR, for 4.4 and 4.5. In case its not appropriate I'll just drop or move to new PR.

**Comment by petr-muller** [comment]
> does it need to be added in this PR/followup or is this something your team does automatically along with fast forward from master into release-specific branches?

@enxebre we are syncing (daily) content from the master files to 4.4 and 4.5 files automatically, you do not need to do anything

**Comment by rgolangh** [comment]
> One more thing I noticed: Most projects in Openshift also use automated merges which you should configure by including your repo in the query here: https://github.com/openshift/release/blob/master/core-services/prow/02_config/_config.yaml#L493 and here: https://github.com/openshift/release/blob/master/core-services/prow/02_config/_config.yaml#L696
> 
> I also saw you have `approvers` in the `OWNERS` file in your repo: you may want to enable the approvers plugin (`/approve` workflow) by adding your repo somewhere around here: https://github.com/openshift/release/blob/master/core-services/prow/02_config/_plugins.yaml#L1929

updated both projects, please take a look

**Pull request merged**